plugins/rbac-backend/src/service/permission-policy.test.ts

```diff
@@ -903,6 +973,7 @@ describe('RBACPermissionPolicy Tests', () => {
         newIdentityResponse(),
       );
       expect(decision.result).toBe(AuthorizeResult.DENY);
+      // verifyAuditLogForNonResourcedPermission('', 'test.resource', 'use', AuthorizeResult.DENY);
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes, I did.